tools/openScienceDataImporter.php

```diff
@@ -0,0 +1,157 @@
+<?php declare(strict_types=1);
+
+require_once 'generic_includes.php';
+require_once 'CandidateImporter.php';
+require_once 'VisitImporter.php';
+require_once 'InstrumentImporter.php';
+
+/**
+ * Minimum number of arguments needed to run this script.
+ *
+ * @var int
+ */
+const NUM_ARGS_REQUIRED = 4;
+
+/*
+ * Expected indices of parameters passed to this script via $argv
+ *
+ * @var int
+ */
+const MAPPING_ARG_INDEX = 2;
+const DATA_ARG_INDEX = 3;
+const EXCLUDED_ARG_INDEX = 4; // Used only for visit label imports
+
+/**
+ * Command line argument for the mode of data importation.
+ */
+const CANDIDATE_IMPORT = 'candidate';
+const INSTRUMENT_IMPORT = 'instrument';
+const VISIT_IMPORT = 'visits';
+
+$usage = <<<USAGE
+Usage: 
+
+To import columns from a table: 
+
+php {$argv[0]} %s <mapping.csv> <data.csv>
+    <mapping.csv>   A CSV file containg columns with OLDPSCID, NEWPSCID, and
+                    NEWCANDID
+    <data.csv>      A CSV file containing columns with OLDPSCID,DATA
+
+
+To import instrument information specifically:
+
+php {$argv[0]} %s <mapping.csv> <data.csv>
+    <mapping.csv>   A CSV file containg columns with OLDPSCID, NEWPSCID, and
+                    NEWCANDID
+    <data.csv>      A CSV file containing columns with instrument data. This
+                    file should be created using the dataExtractor tool. The
```

Comment:
if the `dataExtractor.php` tool is moved to the `exporters` folder, will need to update the description here accordingly